examples/stitches/app/entry.server.tsx

```diff
@@ -11,15 +11,16 @@ export default function handleRequest(
   responseHeaders: Headers,
   remixContext: EntryContext
 ) {
-  const sheet = getCssText();
+  let sheet = getCssText();
 
   let markup = renderToString(
     <ServerStyleContext.Provider value={sheet}>
       <RemixServer context={remixContext} url={request.url} />
     </ServerStyleContext.Provider>
   );
 
-  markup = markup.replace("__STYLES__", sheet);
+  sheet = getCssText();
```

Comment:
The change of `const sheet = getCssText();` is unnecessary, repeating twice `getCssText()`

Reply:
Or 
```ts
  const sheet = getCssText();

  let markup = renderToString(
    <ServerStyleContext.Provider value={sheet}>
      <RemixServer context={remixContext} url={request.url} />
    </ServerStyleContext.Provider>
  );

  const updatedSheet = getCssText();
  markup = markup.replace(/<style id=\"stitches\">.*\<\/style>/g, `<style id="stitches">${updatedSheet}</style>`);
```

Reply:
The later looks good, it was not immediately clear to me why calling it twice 👀 And still not clear though 😅  If `getCssText` is only returning a string, why passing it to the initial markup render? It's not gonna be used during first render anyway, right?
I mean changing this to:
```js
  let markup = renderToString(
    <RemixServer context={remixContext} url={request.url} />
  );

  const cssText = getCssText();
  markup = markup.replace(/<style id=\"stitches\">.*\<\/style>/g, `<style id="stitches">${cssText}</style>`);
```
will work for the resulting markup anyway, right?

Reply:
Good eye. This pattern had been copy/pasted from the `styled-components` example, where it appears the server context isn't necessary at all. I've updated it to be a bit cleaner.